src/Pos/Generator/Block/Mode.hs

```diff
@@ -123,7 +124,7 @@ data BlockGenContext = BlockGenContext
     , bgcSystemStart       :: !Timestamp
     , bgcParams            :: !BlockGenParams
     , bgcDelegation        :: !DelegationVar
-    , bgcGenesisUtxo       :: !GenesisUtxo
+    , bgcGenesisTxpContext :: !GenesisTxpContext
```

Comment:
I am convinced that there are places where `GenesisUtxo` is needed, for example, `Tracking.hs`